backend/src/password/EncryptorUtils.ts

```diff
@@ -1,13 +1,13 @@
 /* eslint-disable @typescript-eslint/no-unsafe-call */
 /* eslint-disable @typescript-eslint/no-unsafe-member-access */
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { User } from '@entity/User'
 import CONFIG from '@/config'
 import LogError from '@/server/LogError'
 import { backendLogger as logger } from '@/server/logger'
-import { User } from '@entity/User'
 import { PasswordEncryptionType } from '@enum/PasswordEncryptionType'
 
-// eslint-disable-next-line @typescript-eslint/no-var-requires
+// eslint-disable-next-line @typescript-eslint/no-var-requires, import/no-commonjs
```

Comment:
Can this be done by an import?

Reply:
I'd like to investigate this in another PR - there was a problem with `import { ... } from ' sodium-native' ` - thats why its required.
I made an Issue: https://github.com/gradido/gradido/issues/2903